README.md

```diff
@@ -21,9 +30,14 @@ on:
   pull_request:
   workflow_dispatch:
 
+permissions: {}
+
 jobs:
   ci:
     name: CI
+    permissions:
+      pull-requests: read
+      contents: read
```

Comment:
- With [no permissions](https://github.com/GuySartorelli/issue-template-playground/actions/runs/10188197950) we get told we need `pull-requests: read`.
- After [adding that](https://github.com/GuySartorelli/issue-template-playground/actions/runs/10188211583), we get told we need `contents:read`
- With [both permissions](https://github.com/GuySartorelli/issue-template-playground/actions/runs/10188217301) we have no permissions errors.

Ignore the fact that CI there is still red - that's just 'cause there's nothing for generate-matrix to output for that repo, and CI doesn't like having no matrix when it's told there will be one.